### DISCUSSION TITLE
Welcome to maskphish Discussions!

### DISCUSSION THREAD

**Discussion by jaykali** [jaykali/maskphish#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by byunak** [comment]
Hi everyone,

I was curious if there is anyone able to manage to use their IP Address instead of shortening a URL since I am using another tool to create a phishing website and I do not port forward it. I just want to use it in my network so ip address would be fine, I was curious if there is any way of using IP address instead of an URL.

Thanks,

**Reply by jaykali** [reply]
**Sorry, due to widespread abuse, we no longer allow linking to hosts by IP address.**

**Reply by sahal-mulki** [reply]
uh @billybonka you could use ngrok to portforward, its EXTREMELY easy
